lib/logstash-output-google_cloud_storage_jars.rb

```diff
@@ -0,0 +1,214 @@
+  require 'com/google/api/grpc/proto-google-cloud-trace-v1/0.6.0/proto-google-cloud-trace-v1-0.6.0.jar'
+  require 'org/hamcrest/hamcrest-core/1.3/hamcrest-core-1.3.jar'
+  require 'com/google/api/grpc/proto-google-cloud-dialogflow-v2beta1/0.6.0/proto-google-cloud-dialogflow-v2beta1-0.6.0.jar'
+  require 'com/google/api/grpc/proto-google-cloud-monitoring-v3/0.6.0/proto-google-cloud-monitoring-v3-0.6.0.jar'
+  require 'com/google/api/grpc/proto-google-cloud-error-reporting-v1beta1/0.6.0/proto-google-cloud-error-reporting-v1beta1-0.6.0.jar'
+  require 'com/google/cloud/google-cloud-monitoring/0.41.0-beta/google-cloud-monitoring-0.41.0-beta.jar'
+  require 'com/google/cloud/google-cloud-spanner/0.41.0-beta/google-cloud-spanner-0.41.0-beta.jar'
+  require 'com/google/api/grpc/proto-google-cloud-spanner-admin-database-v1/0.6.0/proto-google-cloud-spanner-admin-database-v1-0.6.0.jar'
+  require 'com/google/oauth-client/google-oauth-client/1.23.0/google-oauth-client-1.23.0.jar'
+  require 'com/google/api/grpc/proto-google-cloud-speech-v1beta1/0.6.0/proto-google-cloud-speech-v1beta1-0.6.0.jar'
+  require 'com/google/cloud/google-cloud-language/1.23.0/google-cloud-language-1.23.0.jar'
+  require 'com/google/api/grpc/proto-google-cloud-logging-v2/0.6.0/proto-google-cloud-logging-v2-0.6.0.jar'
+  require 'com/google/api/grpc/proto-google-common-protos/1.5.0/proto-google-common-protos-1.5.0.jar'
+  require 'com/google/cloud/google-cloud-compute/0.41.0-alpha/google-cloud-compute-0.41.0-alpha.jar'
+  require 'com/google/api/grpc/grpc-google-common-protos/1.5.0/grpc-google-common-protos-1.5.0.jar'
+  require 'com/google/api/grpc/proto-google-cloud-pubsub-v1/0.6.0/proto-google-cloud-pubsub-v1-0.6.0.jar'
+  require 'com/google/apis/google-api-services-dns/v1-rev66-1.23.0/google-api-services-dns-v1-rev66-1.23.0.jar'
+  require 'com/google/api/grpc/proto-google-iam-v1/0.6.0/proto-google-iam-v1-0.6.0.jar'
+  require 'junit/junit/4.12/junit-4.12.jar'
+  require 'com/google/cloud/google-cloud-core-http/1.23.0/google-cloud-core-http-1.23.0.jar'
+  require 'com/google/cloud/google-cloud-resourcemanager/0.41.0-alpha/google-cloud-resourcemanager-0.41.0-alpha.jar'
+  require 'com/google/api/gax-grpc/1.20.0/gax-grpc-1.20.0.jar'
+  require 'org/apache/httpcomponents/httpcore/4.0.1/httpcore-4.0.1.jar'
+  require 'com/google/errorprone/error_prone_annotations/2.1.2/error_prone_annotations-2.1.2.jar'
+  require 'com/google/api/grpc/proto-google-cloud-bigtable-v2/0.6.0/proto-google-cloud-bigtable-v2-0.6.0.jar'
+  require 'com/google/api/gax-httpjson/0.37.0/gax-httpjson-0.37.0.jar'
+  require 'com/google/cloud/google-cloud/0.41.0-alpha/google-cloud-0.41.0-alpha.jar'
+  require 'com/google/apis/google-api-services-translate/v2-rev47-1.22.0/google-api-services-translate-v2-rev47-1.22.0.jar'
+  require 'com/google/auth/google-auth-library-oauth2-http/0.9.0/google-auth-library-oauth2-http-0.9.0.jar'
```

Comment:
I am flabbergasted by the amount of jars required here. 🤔

Reply:
Good catch! I was using the main gcp JAR for developing, I'll use the GCS specific one.